content/docs/guides/responsive_amp@es.md

```diff
@@ -86,25 +63,44 @@ usando propiedades comunes de CSS. Por ejemplo:
 </body>
 [/sourcecode]
 
-**Importante:**
+{% call callout('Important', type='caution') %}
```

Comment:
For the English word "Important", can you change it to the Spanish word, which I believe is "Importante"?